### PR TITLE
Add HTMLLinkElement support

### PR DESCRIPTION
**Pull request by modulesio**
This is mostly for fixing https://projects.gatunes.com/wsvr/#/41983/144,-144, which is a very 2D site at boot.

### CODE REVIEW

**Comment by modulesio** [comment]
This gets us to everything except drawing the chunks cubemarch. The chunks are generating since you can walk on them,.

capture 9

File: core.js [hunk 1]
```diff
@@ -874,6 +874,24 @@ global._runHtml = (element, window) => {
               });
           }
         }
+      } else if (el instanceof window.HTMLLinkElement) {
+        if (el.run()) {
```
Comment (ngokevin) [inline core.js]: Perhaps this logic can be part of the class as an internal method? Also a comment of what this code does could be useful in the future.

Comment (modulesio) [reply core.js]: Agree with both.

Not actually related to this PR though, so will make it separate.

**Comment by modulesio** [comment]
Tested with `node . -x none https://projects.gatunes.com/wsvr/#/41983/144,-144`, but there's still more to be done on that site.

**Pull request merged**